composer.json

```diff
@@ -7,7 +7,8 @@
 	],
 	"require": {
 		"php": "^7.2 || ^8.0",
-		"phpstan/phpstan": "^1.8.0"
+		"phpstan/phpstan": "^1.8.0",
+		"doctrine/annotations": "^1.11.0"
```

Comment:
Why is this in `require`?

Reply:
I've move this to require because it was used in the constructor but with the other comment you had this can indeed be reverted and the extension will have it as an optional dependency!